dvc/exceptions.py

```diff
@@ -268,6 +268,13 @@ def __init__(self, path, cause=None):
         )
 
 
+class FileOutsideRepoError(DvcException):
+    def __init__(self, path, cause=None):
```

Comment:
Since you are not using it anyway
```suggestion
    def __init__(self, path):
```